src/System.Private.CoreLib/src/System/Environment.UWP.cs

```diff
@@ -46,5 +46,11 @@ public static string MachineName
                 throw new PlatformNotSupportedException();
             }
         }
+
+        public static void Exit(int exitCode)
+        {
+            // Store apps have their lifetime managed by the PLM
+            throw new PlatformNotSupportedException();
```

Comment:
(FailFast will just crash and burn, which likely includes sending Watson reports to the Store developer dashboard, etc.)